lib/monitoring/middleware/conjur_collector.rb

```diff
@@ -0,0 +1,65 @@
+require 'benchmark'
+require ::File.expand_path('../../pub_sub.rb', __FILE__)
+
+module Monitoring
+  module Middleware
+    class ConjurCollector
```

Comment:
Kumbi mentioned this on my PR and I tend to agree - the naming of this class should probably reflect flavor of metrics we are implementing (Prometheus)

Reply:
Once I take your advice below (moving metric init to `prometheus.rb`) the collector shouldn't be beholden to Prometheus. Should be able to trace and publish events with only subscribers needing to know how they're used.